app/src/main/java/ca/etsmtl/applets/etsmobile/ui/activity/MoodleAssignmentsActivity.java

```diff
@@ -348,6 +345,11 @@ public void onChanged(@Nullable RemoteResource<MoodleAssignmentSubmission> moodl
         });
     }
 
+    private void displayErrorMessage(String errorMsg) {
+        Toast toast = Toast.makeText(this, errorMsg, Toast.LENGTH_SHORT);
```

Comment:
Tu pourrais remplacer les deux lignes avec un one liner du genre:
`Toast.makeText(this, errorMsg, Toast.LENGTH_SHORT).show();`

C'est plus concis de cette manière. 😛